ledfx/effects/audio.py

```
@@ -29,6 +29,15 @@
 MAX_MIDI = 108
 
 
+def _stream_stop_worker(stream):
+    """Target for the timeout thread in _close_stream.
+    Calls stream.stop() which may block indefinitely on some drivers."""
+    try:
+        stream.stop()
+    except Exception:
+        pass  # Caller will fall back to abort()
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)